frontend/src/app/modules/boards/board/board-list/board-list.component.ts

```diff
@@ -405,7 +405,9 @@ export class BoardListComponent extends AbstractWidgetComponent implements OnIni
       .subscribe(
         query => this.wpStatesInitialization.updateQuerySpace(query, query.results),
         error => {
-          this.loadingError = this.halNotification.retrieveErrorMessage(error);
+           jQuery('#' + this.queryId).removeClass('boards-list--item');
```

Comment:
The angular way would be to set a variable here and then change the `boards-list--item` class based it: https://angular.io/api/common/NgClass

Reply:
Hi @ulferts , If I define a variable here in board-list.component and use ng-class in board-list-container template, the defined variable is not accessible in board-list-container html file. Because boards-list--item class is outside of board-list component in  board-list-container html file,  i used jquery. What's your idea?

Reply:
@aleixsuau can you please help us out here?

Reply:
State should be contained in the shared parent of all components that need access to it, and then you pass it around via inputs/outputs.